src/extend-utils.ts

```diff
@@ -86,3 +88,42 @@ export function getWalletUtil(name: string, hre: HardhatRuntimeEnvironment) {
     wallet.accountPath = getAccountPath(wallet.accountPath, hre);
     return wallet;
 }
+
```

Comment:
I think DevnetUtils could be in a different file, since it has its own purpose

Reply:
[da02ed0](https://github.com/Shard-Labs/starknet-hardhat-plugin/pull/58/commits/da02ed0b8e4dd3daa4ef5c8b2783dd7aafdc6272)